providers/attio/metadata.go

```diff
@@ -0,0 +1,77 @@
+// nolint
+package attio
+
+import (
+	"context"
+
+	"github.com/amp-labs/connectors/common"
+)
+
+// This struct is used for when the response data having slice of data.
+type responseObject struct {
+	Result []map[string]any `json:"data"`
```

Comment:
I would have named this Data instead of Result.

Reply:
If we specify the Result instead of Data, when we are unmarshalling the response for getting metadata it can't unmarshal because the response data contains "data" as the key.
![image](https://github.com/user-attachments/assets/bdaa7d23-7bf1-444e-8708-35ad534dfae1)

Reply:
I think the suggestion is 
```
Data []map[string]any `json:"data"`
```
if understood correctly.